### PR TITLE
disallowNewlineBeforeBlockStatements: check if prevToken is defined so t...

### PR DESCRIPTION
**Pull request by tteltrab**
...hat files starting with block statement don't throw error

Fixes: #887

### CODE REVIEW

File: lib/rules/disallow-newline-before-block-statements.js [hunk 1]
```diff
@@ -97,7 +97,8 @@ module.exports.prototype = {
             var openingBrace = tokens[openingBracePos];
             var prevToken = tokens[openingBracePos - 1];
 
-            if (openingBrace.loc.start.line !== prevToken.loc.start.line) {
+            if (typeof prevToken === 'object' &&
```
Comment (zxqfox) [inline lib/rules/disallow-newline-before-block-statements.js]: Nice catch & thanks for contributing!

Probably better to use here [new assertions](https://github.com/jscs-dev/node-jscs/blob/master/lib/token-assert.js#L81). Something like:
```
var openingBrace = file.getFirstNodeToken(node);
var prevToken = file.getPrevToken(openingBrace);
if (!prevToken) {
  return;
}

errors.assert.sameLine...
```

**Comment by tteltrab** [comment]
ah, cool.  didn't know about those, but will do - could use an update.  And the strict type check is probably overkill, wasn't sure which way was preferred.

**Comment by zxqfox** [comment]
@tteltrab Definitely ;-D

**Comment by tteltrab** [comment]
Pull request is #888 I believe.

**Pull request closed without merging**